### DISCUSSION TITLE
'Next' button in questionnaire does not work

### DISCUSSION THREAD

**Discussion by erwindegrave** [globaleaks/globaleaks-whistleblowing-software#3417] [General]
Hi,

I have created a questionnaire with 4 steps, but the 'Next' button does not work. The 'Previous' button does work.

Thanks.

**Comment by evilaliv3** [comment]
Thank you for your question.

We are not aware of bugs on the current version of the software.

Which version are you using?

**Comment by erwindegrave** [comment]
I am using the latest version 4.11.4.

What other information do you need from me?

**Comment by erwindegrave** [comment]
At the end I solved this issue. It is actually very simple one... I did not have a recipient assigned to the questionnaire, and that is why the next button did not work :)
